src/altinn-app-frontend/src/components/summary/SummaryComponent.tsx

```diff
@@ -150,48 +148,46 @@ export function SummaryComponent(props: ISummaryComponent) {
     if (!formComponent) {
       return null;
     }
-
-    switch (formComponent.type) {
-      case 'Group':
+    const change = {
+      onChangeClick,
+      changeText
+    }
+    switch (formComponent.type.toLowerCase()) {
       case 'group': {
         return (
           <SummaryGroupComponent
-            onChangeClick={onChangeClick}
-            changeText={changeText}
+            {...change}
             {...props}
           />
         );
       }
-      case 'FileUpload': {
+      case 'fileupload': {
```

Comment:
Not sure we can change the casings here - this is case sensitive, isnt it?

Reply:
On line 155 i set the formComponent.type.toLowerCase().
I also removed case `Group` as it seems that having a case for every individual casing possible seems a bit messy to me.

Reply:
Aha, I missed that part... But is it a good idea to change this? Then it will be out of sync with casing from formlayout.

An even better approach would be to use an enum for these instead I think. Then we would also get an error if something is misspelled.